Mapcycle Update

mehwaffle10 opened the pull request:
## Status

Work-in-Progress

## Description

Remake of map pools for all game modes. Includes maps from the current pool, old maps previously in the pool, and new maps submitted in discord. Also reorganizes the map folder structure to be much more streamlined. in `Base/Maps` there is a folder per gamemode. All maps in the top level of their gamemode directory is currently in the pool. There are also `Dormant` and `Retired` folders in each gamemode's folder for maps that are not currently in the pool and maps that should never be added to the pool. Removes duplicate maps in different directories or multiple versions of the same map

mugg91 commented:
`Coroz_floating_rifts` has flowers on stone tiles that don't grow.

mugg91 commented:
I guess it is nice that someone is cleaning up maps. Although I don't like cropping on some of the maps (e.g. `Ferrezinhre_Vertigo`). Mappers made some nice designs and playing games should also be about discovering things, like it makes the map feel more alive if you know what I mean. I don't think any of the maps were that big that players were suffering slowdowns and cropping maps doesn't really save that much file size.

mehwaffle10 commented:
Files are trimmed to make maps load faster and less load intensive on weaker clients, not to reduce file size. Very impactful with how often maps are loaded in TDM. I have done what I can to maintain as much detailing as possible while optimizing. I do what I can not to impact gameplay at all either. The only thing cut for detailing in vertigo is some small dirt protrusions that hardly anyone would notice. Plus now it looks like a hamburger which is a W in my books

mugg91 commented:
Maybe you can incorporate https://github.com/transhumandesign/kag-base/pull/1823 to fix https://github.com/transhumandesign/kag-base/issues/1813

mehwaffle10 committed:
Update Monsteri_CTF1.png

Set red barrier

mehwaffle10 committed:
Monsteri_CTF2

Raise bedrock substantially, moving spawn cave to accommodate. Increase map symmetry. Reduce tunnel spots. Raise map slightly. Adjust resources

mehwaffle10 committed:
Monsteri_CTF3

Raise bedrock substantially. Remove void under lakes. Make map symmetrical. Set red barrier. Adjust resources heavily. Shorten map height. Reduce tunnel spots

mehwaffle10 committed:
Update Monsteri_CTF3.png

Lower water level by one. Fill in holes under flag

mehwaffle10 committed:
Update Snatchmark.png

Lower map height. Raise bedrock and trim excess

mehwaffle10 committed:
Update TresFlags.png

Make map symmetrical. Lower map height. Raise bedrock drastically and add bedrock to islands. Adjust resources. Add greenery

mehwaffle10 committed:
Update TresFlags.png

Bushes

mehwaffle10 committed:
Update NewTortuga.png

Add more stone to mid

mehwaffle10 committed:
Update Cleaf_Explorersground.png

Raise bedrock substantially around mid. Trim excess bedrock. Shorten map height drastically. Adjust resources. Set red barrier

mehwaffle10 committed:
Update Cleaf_Trumble.png

Trim excess bedrock

mehwaffle10 committed:
Update Coroz_floating_rifts.png

Trim map height sleightly

mehwaffle10 committed:
Update Corroding_plains.png

Raise bedrock substantially and trim excess. Add more bedrock. Reduce strong tunnel spot potential. Adjust tree locations. Move second tree above tent. Lower map height drastically

mehwaffle10 committed:
Ej_Map

Rename to follow naming convention

mehwaffle10 committed:
Update Ej_BlueSunset.png

Raise bedrock drastically and trim excess. Raise map height. Adjust stone and trees. Prevent overly strong tunnel spots

mehwaffle10 committed:
Update Ej_Cataracts.png

Raise bedrock *dramatically*. Adjust resources accordingly. Lower map height. Add greenery. Make section in front of flag floodable from mid

mehwaffle10 committed:
Update Ej_Chambers.png

Lower map height slightly

mehwaffle10 committed:
Update Ferrezinhre_Alchemy.png

Raise bedrock and make not a void map. Adjust resources. Lower map height. Remove bottom flag. Add greenery. Add some slight bedrock

mehwaffle10 committed:
Update Ferrezinhre_Aojiroku.png

Add greenery, fixing some visuals with grass in weird places. Make the floating islands under mid thickstone. Adjust gold. Shorten map height

mehwaffle10 committed:
Update Ferrezinhre_Asuron.png

Reduce crazy tunnel spots. Adjust resources. Thin ceiling in general. Remove second flag. Raise mid to be level with flag. Remove random void potential. Raise ceiling above mid and flag. Add greenery and remove weird dirt backwall artifacts. Thin dirt backwall at mid. Add prebuilt arches at mid

mehwaffle10 committed:
Update Ferrezinhre_Aurochs.png

Raise bedrock. Reduce flooding potential. Lower map height

mehwaffle10 committed:
Update Ferrezinhre_Aurochs.png

Add greenery

mehwaffle10 committed:
Update Ferrezinhre_Aurochs.png

Bedrock under flag

mehwaffle10 committed:
Update Ferrezinhre_Avinaaq.png

Remove void potential and raise bedrock. Pull back red barrier. Adjust resources. Add greenery. Lower map height

mehwaffle10 committed:
Update Ferrezinhre_Dashune_Pass.png

Lower map height. Adjust resources. Remove two back flags. Lower front flag

mehwaffle10 committed:
Update Ferrezinhre_Echo_Dissociation.png

Add stone. Add greenery

mehwaffle10 committed:
Update Ferrezinhre_Echo_No2_Replica.png

Raise mid and subsequently bedrock. Adjust red barrier and resources. Add greenery

mehwaffle10 committed:
Update Ferrezinhre_Fagra.png

Lower map height. Lower top flag. Remove bottom flag. Adjust resources. Raise bedrock to reduce already ratty map. Add greenery

mehwaffle10 committed:
Ferrezinhre_Fragra

Rename to avoid potential slur confusion

mehwaffle10 committed:
Monsteri_Plunder

Rename to Monster_Plunder, mainly so that aseprite doesn't think all of his maps are part of the same spritesheet. Fix grass growing on stone

mehwaffle10 committed:
Monsteri_ChickenIsland

Rename to Monster_ChickenIsland, mainly so that aseprite doesn't think all of his maps are part of the same spritesheet (I've also always called this map chicken island). Fix grass growing on stone

mehwaffle10 committed:
Monsteri_Sharcave

Rename to Monster_Sharcave, mainly so that aseprite doesn't think all of his maps are part of the same spritesheet

mehwaffle10 committed:
Update Monsteri_Sharcave.png

Fix trimming

mehwaffle10 committed:
Update Ferrezinhre_Fragment.png

Remove top mid structure. Adjust resources. Adjust bedrock. Make burnable paths in mid. Lower map height drastically

mehwaffle10 committed:
Update Ferrezinhre_Genashi.png

Lower map height drastically. Fill in tunnel pits and raise bedrock. Adjust resources. Remove back flag. Pull front flag and red barrier slightly back. Add greenery

mehwaffle10 committed:
Update Ferrezinhre_Highlands.png

Lower map height. Raise bedrock. Set red barrier. Adjust resources. Add greenery

mehwaffle10 committed:
Update Ferrezinhre_Miroi.png

Lower map height drastically. Remove giant rat tunnels. Raise mid and remove void pits. Lower cliffs around tent. Remove back flag. Push top flag and red barrier forward. Adjust resources. Add greenery. Add bedrock to mid. Make dirt backwall less sporatic

mehwaffle10 committed:
Update Ferrezinhre_Miroi.png

Raise map height slightly

mehwaffle10 committed:
Update Ferrezinhre_Mithilorn.png

Literally cut the map more than in half. Reduce map height drastically. Adjust resources. Add bedrock. Add greenery. Remove back flag

mehwaffle10 committed:
Update Ferrezinhre_Moonfall.png

Fix teams being on the wrong side. Remove back flag. Lower map height dramatically. Raise and add bedrock. Adjust resources. Add greenery. Push red barrier forward slightly

mehwaffle10 committed:
Update Ferrezinhre_Multi_Level_Crossing.png

Adjust resources. Add greenery. Lower map height slightly

mehwaffle10 committed:
Update Ferrezinhre_Oduvtaar.png

Remove spawn cave. Raise bedrock. Lower map height drastically. Adjust resources. Add greenery. Remove back flag. Pull red barrier back

mehwaffle10 committed:
Update Ferrezinhre_Oduvtaar.png

Raise map height slightly

mehwaffle10 committed:
Update Ferrezinhre_Oduvtaar.png

Thin bedrock on flag slightly

mehwaffle10 committed:
Ferrezinhre_Onori

Requires too drastic changes that would likely still not be enjoyable in vanilla balance. Retiring

mehwaffle10 committed:
Update Ferrezinhre_Voice_Universal_Nemesis.png

Adjust resources, dirt backwall, bedrock, and greenery. Lower map height slightly

mehwaffle10 committed:
Update Ferrezinhre_Voltaric.png

Raise bedrock. Adjust resources. Add greenery. Add bedrock

mehwaffle10 committed:
Update Ferrezinhre_We_will_kiss_in_the_moonlight.png

Remove excess gold in ceiling. Pull red barrier back. Remove back flag. Add greenery

mehwaffle10 committed:
Update ffsff_PlatformFlats.png

Fix some grass

mehwaffle10 committed:
Update ffsff_SecludedForest.png

Reduce map height. Raise bedrock. Reduce excess gold, stone and trees. Move all trees to mid to reinforce the forest motif. Lower cliffs and tent to allow for lower map height. Set red barrier. Add greenery

mehwaffle10 committed:
Update FG_99.png

Trim excess bedrock. Add greenery

mehwaffle10 committed:
Update FG_Fallout.png

Reduce map height. Raise bedrock. Adjust resources. Add greenery. Adjust red barrier and flag bedrock

mehwaffle10 committed:
Update FG_Forelands.png

Adjust greenery

mehwaffle10 committed:
Update FG_Forsaken.png

Add greenery

mehwaffle10 committed:
Update FG_FourLakes.png

Add greenery

mehwaffle10 committed:
Update FG_Sinkhole.png

Add greenery

mehwaffle10 committed:
Update Fuzzle_Stale.png

Lower map height. Add greenery

mehwaffle10 committed:
Update Gurin_DoubleSided.png

Adjust resources. Shorten map height. Raise bedrock. Make symmetrical

mehwaffle10 committed:
Update GuyDeLu_Dragons_Den.png

Add greenery. Raise map height slightly

mehwaffle10 committed:
Update hugobh_Grottolands.png

Reduce map height. Raise bedrock. Remove void potential. Lower flag and top mid. Add dirt structure in front of flag. Remove water cave under flag. Adjust resources. Add greenery

mehwaffle10 committed:
Update JTG_Aqueduct.png

Raise bedrock. Remove super powerful tunnel positions. Lower map height. Reduce tree count. Distribute gold around mid. Have mid open to lower tunnel at the get go. Thin some dirt backwall in mid and in tunnel. Add greenery

mehwaffle10 committed:
Update JTG_BoulderLake.png

Set red barrier past second flag. Raise bedrock substantially. Adjust resources. Greatly reduce flood potential. Shorten map height. Add greenery

mehwaffle10 committed:
Update JTG_FirstBadLands.png

Make symmetrical. Set red barrier. Adjust resources and bedrock. Shorten map height

mehwaffle10 committed:
Update Lavaguy678_Meadow.png

Lower map height. Adjust red barrier. Adjust greenery

mehwaffle10 committed:
Update Magmus_hangingmus.png

Lower map height. Adjust red barrier

mehwaffle10 committed:
Update Magmus_Holegmus.png

Adjust red barrier. Reduce map height. Adjust greenery

mehwaffle10 committed:
Update Mazey_Epic.png

Adjust red barrier

mehwaffle10 committed:
Update Mazey_Junctions.png

Adjust red barrier. Lower map height. Raise bedrock. Adjust resources. Widen bottom tunnel

mehwaffle10 committed:
Update Mazey_Rioter.png

Adjust greenery. Adjust red barrier

mehwaffle10 committed:
Update Mazey_Rioter.png

grass

mehwaffle10 committed:
Update mcrifel_fish.png

Adjust greenery. Lower map height. Set red barrier

mehwaffle10 committed:
Update mcrifel_Steppes.png

Lower map height. Adjust greenery. Set red barrier

mehwaffle10 committed:
Update Ni_Contingency.png

Raise bedrock. Lower map height. Adjust resources. Reduce tunnel spots. Thin dirt backwall. Make sure there's no dirt backwall under the stone backwall in mid. Widen lower path. Pull red barrier closer to flag. Add bedrock to dirt islands

mehwaffle10 committed:
Update Oodle_Remnant.png

Raise bedrock. Lower map height. Set red barrier. Add bedrock to islands. Make map more symmetrical. Adjust resources. Reduce dirt backwall. Widen lower passages. Add greenery. Reduce flooding potential

mehwaffle10 committed:
Oodle_Stonethrow

Revert to previous smaller form. Adjust resources. Raise bedrock. Add greenery. Move to SmallCTF instead of CTF

mehwaffle10 committed:
Update Potatobird_CubeCanyon.png

Remove void pits. Remove back flag. Adjust resources. Set red barrier. Add greenery. Raise map height slightly

mehwaffle10 committed:
Update petgreendino_bisonbisoff.png

Lower map height slightly. Remove most dirt backwall under mid to prevent blocking the bison as easily. Revert to one bison in the center. Make odd width for true middle, add one gold to make give a little bit of wiggle room for 8 gold structures

mehwaffle10 committed:
Punk123 to PUNK123, PUNK123_AlienFrogAssault

Rename PUNK123's maps in accordance with his actual username. For PUNK123_AlienFrogAssault, Raise bedrock drastically. Lower map height drastically. Adjust resources somewhat heavily. Set red barrier. Remove top flag. Add a ton of greenery

mehwaffle10 committed:
Update PUNK123_AntKing.png

Raise bedrock substantially. Raise map height. Remove dirt backwall in corners. Move tent back. Remove top flag. Set red barrier. Move flag islands back. Adjust resources

mehwaffle10 committed:
Update PUNK123_AntQueen.png

Raise bedrock. Adjust resources. Push flag back. Add bedrock to mid to prevent deterioration. Make rat tunnels much more visible and a bit wider. Replace scattered stone with deliberate stone pillars in rat tunnels. Add greenery

mehwaffle10 committed:
Update PUNK123_BlockCity.png

Widen bottom path substantially. Raise bedrock. Add bedrock to mid. Remove flood potential. Adjust resources. Add greenery. Remove back flag. Raise front flag. Set red barrier

mehwaffle10 committed:
Update PUNK123_ChasmSpasm.png

Remove extreme excess amount of dirt. Raise bedrock. Adjust resources. Add greenery. Expand stone chambers at mid. Adjust red barrier

mehwaffle10 committed:
Update PUNK123_CryingDemon.png

Raise bedrock substantially. Remove flood potential. Thin backwall under mid. Move flag to mid island. Squish mid vertically. Stretch entire map horizontally. Adjust resources. Set red barrier

mehwaffle10 committed:
Update PUNK123_Hillo.png

Raise bedrock. Remove front flag. Add bedrock to upper section. Add bridge with bedrock between mid islands. Set red barrier. Adjust resources. Shorten map height. Add greenery

mehwaffle10 committed:
Update PUNK123_MemoryLane.png

Adjust resources. Lower map height slightly. Remove a tunnel spot. Adjust red barrier

mehwaffle10 committed:
Update PUNK123_Nubtytown.png

Raise bedrock, remove void pits. Lower map height. Adjust resources. Add greenery. Set red barrier. Add more room between tent and flag. Remove flood potential

mehwaffle10 committed:
Update PUNK123_CryingDemon.png

Add bedrock

mehwaffle10 committed:
Update PUNK123_Hillo.png

Add bedrock

mehwaffle10 committed:
Update PUNK123_Nubtytown.png

Fix red flag and tent

mehwaffle10 committed:
Update PUNK123_PlatformedPredicament.png

Raise bedrock. Shorten map height. Adjust resources. Add bedrock to platforms. Move tent away from flag

mehwaffle10 committed:
Update PUNK123_PrincessCity.png

Raise bedrock. Lower map height. Adjust resources. Add bedrock to mid. Add greenery

mehwaffle10 committed:
Delete PUNK123_PrincessCity2.png

Remove duplicate

mehwaffle10 committed:
Update PUNK123_Rally.png

Lower map height. Adjust resources. Adjust red zone

mehwaffle10 committed:
Update PUNK123_Shark_Rift.png

Fix incorrect color for dirt leading to floating grass

mehwaffle10 committed:
Update CerberosRift.png

Adjust red barrier

mehwaffle10 committed:
Update PUNK123_Shark_Rift.png

Adjust greenery

mehwaffle10 committed:
Update CerberosRift.png

Adjust greenery

mehwaffle10 committed:
Update PUNK123_Shark_Rift.png

Raise bedrock and central island substantially. Raise shark mouths. Adjust resources. Adjust red barrier. Add greenery. Adjust bedrock in shark

mehwaffle10 committed:
Update PUNK123_SkinnedCastle.png

Shorten map height. Pull flag and red barrier back. Add greenery. Adjust resources

mehwaffle10 committed:
Update Radioactive_Dahill.png

Raise bedrock substantially. Lower map height. Set red barrier. Adjust resources. Add greenery. Moved kegs from corners to mid

mehwaffle10 committed:
Update Radioactive_Dahill.png

Adjust bedrock

mehwaffle10 committed:
Update Rayne_KingArthursPonds.png

Raise bedrock and shorten map height dramatically. Lower and push flag forward. Set red barrier. Add bedrock. Adjust resources

mehwaffle10 committed:
Rayne_KingArthursPond

Make dormant especially with Rayne_KingArthursPonds already in the pool

mehwaffle10 committed:
Update Rayne_NerysRidge.png

Lower map height. Raise bedrock drastically. Adjust resources. Add greenery. Set red barrier. Move back flag forward

mehwaffle10 committed:
Update Shadowor_IslandCoast.png

Raise bedrock. Adjust map height. Set red barrier. Widen lake, reduce flood potential. Adjust resources. Add greenery. Redesign mid structure

mehwaffle10 committed:
Shadowor_Minotaur

Convert to Small CTF. Move back flag above front flag. Move tent to old back flag location. Trim sides of maps. Adjust resources. Raise bedrock. Raise map slightly. Add greenery. Set red barrier

mehwaffle10 committed:
Update Shadowor_ProvingGrounds.png

Raise bedrock drastically. Slightly raise map height. Raise mid relative to rest of map. Adjust resources. Add bedrock. Fix flooding issue. Add greenery. Set red barrier

mehwaffle10 committed:
Update SJD360_Stampede.png

Remove premade rat tunnels. Raise bedrock substantially. Make symmetrical. Clean up and open mid. Clean up spawn wood structure. Lower map height. Set red barrier

mehwaffle10 committed:
Update SJD360_Abrogation.png

Raise bedrock drastically. Raise lower mid and cave system. Adjust resources. Lower map height substantially. Add greenery. Set red barrier. Make symmetrical

mehwaffle10 committed:
Update Skinney_Crypt.png

Lower map height. Add greenery. Adjust bedrock

mehwaffle10 committed:
Update Skinney_Descent.png

Lower map height slightly. Add bedrock. Add greenery

mehwaffle10 committed:
Update Skinney_Glitch.png

Adjust greenery

mehwaffle10 committed:
Update Skinney_Bluff.png

Lower map height. Remove tunnel spot. Set red barrier. Add slightly more bedrock to mid and make bedrock symmetrical. Add greenery

mehwaffle10 committed:
Update Skinney_Descent.png

Adjust red barrier

mehwaffle10 committed:
Update Skinney_Glitch.png

Adjust red barrier. Add greenery

mehwaffle10 committed:
Update Skinney_Typhoon.png

Lower map height. Raise bedrock. Set red barrier. Adjust resources. Add greenery

mehwaffle10 committed:
Update Trekobius_DragonsLair.png

Raise bedrock. Lower map height drastically. Adjust resources. Add greenery. Set red barrier

mehwaffle10 committed:
Biurza_Adapt_Overcome

Unretire. Reduce map height drastically. Reduce valley slope. Raise and adjust bedrock. Add greenery

mehwaffle10 committed:
Ferrezinhre_Golden_Melancholy

Remove back flag. Shorten map height. Reduce gap sizes across the board. Add greenery (treeeeeeeeeeees). Add bedrock to islands. Reduce pit potential. Adjust resources

mehwaffle10 committed:
Joiken_Ionic

Rename from CTF2, named after Greek Ionic columns. Raise bedrock substantially. Remove back flag. Lower map height. Reduce hill size. Set red barrier. Add greenery

mehwaffle10 committed:
an_obamanation_AlienAntAssault

Remove from dormant. Raise bedrock. Lower map height. Move flags together somewhere in middle between two flag locations. Set red barrier. Adjust resources. Add greenery. Fix typo in name

mehwaffle10 committed:
bunnie_Elantris

Remove from dormant. Reduce gap between flag island and mid. Remove back flag. Adjust bedrock. Lower map height drastically. Add greenery

mehwaffle10 committed:
Update FG_Rocktail.png

Replace mid wood blocks with platforms so it's easier to bounce up off the trampoline

mehwaffle10 committed:
Red Barrier Changes

Adjust red barrier on all maps since we can no longer build in red barrier. Adjust greenery on some maps. Adjust some dirtbackwall on some maps

mehwaffle10 committed:
Red Barrier Changes Round Two

Finish adjusting red barriers and greenery the rest of the CTF maps that are currently in the pool

mehwaffle10 committed:
Update mapcycle.cfg

Current draft

mehwaffle10 committed:
Update BleakArena.png

Adjust water timer. Adjust visuals. Reduce flooding. Crush some sections horizontally or vertically

mehwaffle10 committed:
Update BleakArena.png

Adjust jumps

mehwaffle10 committed:
Update Biurza_Garbage_Collector.png

Remove drill since theyre buggy

mehwaffle10 committed:
Update Biurza_Trench_Punk.png

Adjust movement

mehwaffle10 committed:
Update Biurza_Trench_Punk.png

Adjust movement

mehwaffle10 committed:
Small CTF Red Barriers

Adjust red barriers in small CTF since you can no longer build in them. Adjust greenery. Fix wrong sides

mehwaffle10 committed:
Small CTF Red Barrier

Final pass of updating red barriers and greenery for small CTF. Update mapcycle.cfg

mehwaffle10 committed:
New Maps

Add approved maps to the pool, adjusting red barrier and greenery

mehwaffle10 committed:
Update Biurza_Tramp_Trap.png

Squish map horizontally and vertically. Adjust movement

mehwaffle10 committed:
Update bunnie_Wojciech_Jaruzelski.png

Squish map horizontally. Add ceiling. Add void pits on edge of map. Make chest rooms all wood and only outwards facing platforms. Move shops to single mid shop

mehwaffle10 committed:
Update Cruxiat_totem.png

Crush map vertically, flatten sections. Adjust movement. Water column on bottom floor

mehwaffle10 committed:
Update Eluded_JoansMansion.png

Overhaul visuals. Add mechanism door. Make pit more of a focal point. Add trampoline. Adjust movement

mehwaffle10 committed:
Update Ferrezinhre_CursedWell.png

Raise platforms by spawn 1 tile. Make symmetrical

mehwaffle10 committed:
Update Ferrezinhre_Sundered.png

Squish horizontally and vertically. Remove edges. Adjust visuals. Add shop under mid between arrow traps. Add chest on one wide platform in bottom of arrow pits. Widen pits

mehwaffle10 committed:
Update FG_Box.png

enhance the box

mehwaffle10 committed:
Update FG_Clash.png

Add way to climb up platforms in mid

mehwaffle10 committed:
Update FG_Collide.png

Remove outer edges. Crush map vertically. Adjust movement

mehwaffle10 committed:
Update FG_Enclosure.png

Rework bottom area to pit traps with mechanisms. Adjust movement. Remove water. Add button for top dispensers

mehwaffle10 committed:
Update TheCauldron.png

Make trees symmetrical

mehwaffle10 committed:
Update bunnie_Korwin_Mikke.png

Crush map horizontally. Adjust collapses. Remove door mechanisms. Move shops. Add platforms for corner jumps. Move dispensers to ceiling so they do not interrupt shield slides. Streamline outer structures

mehwaffle10 committed:
Map Names

Rename maps to not just be "map" or "<gamemode>". Mostly so Aseprite doesn't think they're part of the same sprite sheet but also to make them more memorable

mehwaffle10 committed:
Update mapcycle.cfg

Missing in last commit

mehwaffle10 committed:
Update 8x_DoubleTrouble.png

Make symmetrical. Improve movement greatly. Add traps to drop mines. Make mines neutral. Make pits more relevant. Adjust fire potential

mehwaffle10 committed:
Trees

Make easier to see

mehwaffle10 committed:
bunnie_SkyAlchemy

Take out of dormant. Lower map height. Adjust greenery. Add bedrock. Remove bottom flag. Raise mid above flag height. Adjust resources

mehwaffle10 committed:
Fellere_FourRivers_Adjusted

Remove from dormant. Heavily adjust resources. Heavily lower map height and raise bedrock. Lower flag and mid heights. Smooth out tunnels. Adjust flooding. Flatten cliffs at edge of map. Adjust red barrier

mehwaffle10 committed:
Gix_Devastated_Lake

Removed back flag. Lower map height drastically. Raise bedrock drastically. Raise mid lakes to be more level with flag lakes. Adjust resources heavily. Make mid lake less deep

mehwaffle10 committed:
4zK_Rorschach

Remove from retired. Flatten map. Push top flag forward. Remove water and chest. Adjust bedrock and resources

mehwaffle10 committed:
Hugobh_King_Gallery

Crush map vertically and horizontally. Clean up visuals. Flatten cliffs. Reduce flooding potential. Add greenery. Adjust resources. Adjust red barrier. Shorten map height. Raise bedrock. Update mapcycle

mehwaffle10 committed:
Update mapcycle.cfg

mehwaffle10 committed:
Greenery

Clean up and add greenery on a bunch of maps. Make trees 5 tall to make them more visible. Adjust visuals. Raise spawns out of ground where possible

mehwaffle10 committed:
Greenery 2

Finish previous commit

mehwaffle10 committed:
Update 8x_Cascade.png

Fix sunk spawns

mehwaffle10 committed:
Update NoahTheLegend_OuterExpanse.png

small cleanup

mehwaffle10 committed:
Sunk Spawns

Fix sunk spawns where appropriate

mehwaffle10 committed:
Update 4zK_Rorschach.png

Naked dirt on ceiling

mehwaffle10 committed:
Biurza_UFO

Unretire. Shorten map vertically. Flatten map. Adjust bedrock and resources. Add greenery. Add ship lights and abducted bovine

mehwaffle10 committed:
bunnie_Dellutha

Unretire. Lower map height. Cut bottom section of map off. Raise bedrock. Adjust flooding. Add greenery. Adjust resources. Remove back flag. Crush horizontally

mehwaffle10 committed:
petgreendino_skyson

Remove from retired. Remove gap between islands. Add bedrock and greenery. Adjust resources. Make bottom path more viable. Squish map vertically

mehwaffle10 committed:
Redshadow6_twinlakes

Remove from retirement. Raise bedrock and lower map height. Adjust resources. Squish map horizontally. Make map more floodable/boatable. Widen mid tunnel height. Adjust dirt backwall. Add greenery

mehwaffle10 committed:
SJD360_Overdose

Remove from retirement. Crush map horizontally. Change to 1 flag at edge of cave. Raise cave to be more level with ground height. Move tent to back of spawn structure. Adjust resources heavily. Raise bedrock heavily. Thin dirt backwall. Lower map height. Thin mid vertically. Add greenery

mehwaffle10 committed:
SJD360_Paradox

Remove from retirement. Remove tunnel system entirely. Raise bedrock. Adjust island relative heights. Heavily adjust resources. Add greenery. Set red barrier. Squish map horizontally. Make symmetrical

mehwaffle10 committed:
Create Basshunter_ratsndragons.png

New map. Heavily revised version of original submission. See thread for info

mehwaffle10 committed:
Create kraken_commerce.png

New map. See thread for details

mehwaffle10 committed:
New Maps

Gingerbeard and redshadow6 maps. Check respective threads in discord for info

mehwaffle10 committed:
Create redshadow6_forsaken.png

New map. Check discord thread for details

mehwaffle10 committed:
Create redshadow6_mausoleum.png

New map. See discord thread for deatails

mehwaffle10 committed:
Create redshadow6_mirkwood.png

Initial commit. See Discord thread for details

mehwaffle10 committed:
Update redshadow6_mirkwood.png

Greenery

mehwaffle10 committed:
Create redshadow6_ShangriLa.png

New map. See Discord thread for details

mehwaffle10 committed:
Create redshadow6_sharknest.png

New map. See Discord thread for details

mehwaffle10 committed:
Update FG_SpaceInvader.png

Remove platforms on upper level so crouch slides aren't interrupted

mehwaffle10 committed:
Update Ferrezinhre_Still_Free.png

Add platforms for corner jumps

mehwaffle10 committed:
Create Japodis1_HillSides.png

New map. Check Discord thread for details

mehwaffle10 committed:
Create slepovnick_n_BridgeBetweenNations.png

New map. Check Discord thread for details

mehwaffle10 committed:
Create TerminalHash_Ancient_Towers.png

New map. Check Discord thread for details

mehwaffle10 committed:
Create TerminalHash_Annoying_Job.png

New map. Check Discord thread for details

mehwaffle10 committed:
Create TerminalHash_Bad_Mouse.png

New map. Check Discord thread for details

mehwaffle10 committed:
Create TerminalHash_Glass_Of_Water_Please.png

New map. Check Discord thread for details

mehwaffle10 committed:
CTF Mapcycle

Upadte to include new maps

mehwaffle10 committed:
Update FG_Descent.png

Fix fireplaces clipping

mehwaffle10 committed:
Update Eluded_JoansMansion.png

Streamline movement next to spawn

mehwaffle10 committed:
Update FG_Break.png

Remove one layer of doors so archers can actually fight at top mid

mehwaffle10 committed:
Update kodysch_Upshoot.png

Make bottom more comfortable to move on

mehwaffle10 committed:
Update Lavalord_Serpent.png

Clean up dirt backwall holes

mehwaffle10 committed:
Update MM_Amarok.png

Reduce spawn height. Make tunnel easier to get in. Clean up movement on lower section. Fix shop asymmetry

mehwaffle10 committed:
Update Ni_Ashes.png

Add burgers that float up when flooding

mehwaffle10 committed:
Update NoahTheLegend_OuterExpanse.png

Remove a stone block on the left to improve movement

mehwaffle10 committed:
Update NoahTheLegend_Underhang.png

Add fire arrow dispensers. Make mid platform all wood. Make some of the bottom platforms collapsable

mehwaffle10 committed:
Update PressureCooker.png

Add central path to increase movement paths drastically

mehwaffle10 committed:
Update SJD360_CrowbarsSwitchblades.png

Streamline movement substantially, reducing 1 wide platforms. Add trampolines and flatten bottom

mehwaffle10 committed:
Update slepovnickn_UnderWaterBase.png

Enhance decorations, adding more land mass and making the lighthouse less phallic. Add lighthouse self destruct button

mehwaffle10 committed:
Update slepovnickn_UnderWaterBase.png

Adjust movement on mid/bottom mid

mehwaffle10 committed:
Update MM_Amarok.png

Add platform by trampoline. Change block location for corner jump on top mid

mehwaffle10 committed:
Basshunter_UndergroundPool

New map. See discord thread for details

mehwaffle10 committed:
Oodle Rename

Rename Oodle to kodysch to keep in line with username convention

mehwaffle10 committed:
Update TheCauldron.png

grass

mehwaffle10 committed:
Create RenderX_Islands.png

New map. See discord thread for details

mehwaffle10 committed:
slepovnickn_BridgeBetweenNations

Fix username

mehwaffle10 committed:
Create slepovnickn_Amirani.png

New map. See discord thread for details

mehwaffle10 committed:
Create slepovnickn_Enfumades.png

New map. Check discord thread for info

mehwaffle10 committed:
Create slepovnickn_FieldsOfGyrtona.png

New map. Check discord thread for info

mehwaffle10 committed:
Create slepovnickn_WorldCore.png

New map. Check discord thread for details

mehwaffle10 committed:
Create WalidZzzz_Ritix.png

New map. Check Discord thread for details

mehwaffle10 committed:
Map Cycles

Update map cycles to include new maps

mehwaffle10 committed:
Oodle Rename

Use username

mehwaffle10 committed:
Update Biurza_NastySewers.png

Raise map height slightly

mehwaffle10 committed:
Update NoahTheLegend_TwoWayStreet.png

Lower map height. Trim excess bottom of map

mehwaffle10 committed:
Update QuantalJ_DrillRat.png

Raise map height

mehwaffle10 committed:
Update redshadow6_pinnacle.png

Shorten mid and map height

mehwaffle10 committed:
Update slepovnickn_DrainedOcean.png

Lower map height

mehwaffle10 committed:
Update slepovnickn_BattleOfStirlingBridge.png

Fix grass

mehwaffle10 committed:
Small CTF Cave Maps

Adjust bedrock for the build height changes

mehwaffle10 committed:
CTF Cave Maps

Adjust cave maps in CTF for map ceiling changes

mehwaffle10 committed:
Hugobh_Grottolands

Fix missing capital letter

mehwaffle10 committed:
Create bunnie_courouz.png

New map. Taken from competitive play

mehwaffle10 committed:
Create burger.png

New map. Taken from competitive play

mehwaffle10 committed:
Create OldTortuga.png

New(?) map. Taken from competitive play. Fairly certain this is actually a super old map

mehwaffle10 committed:
Create Didgedy_TheHillsHaveEyes1.png

New map. Taken from competitive play

mehwaffle10 committed:
Create FG_Watermelon.png

New map. Taken from competitive play

mehwaffle10 committed:
Create mehwaffle10_President.png

New map. Taken from competitive play

mehwaffle10 committed:
Create NKking_lyrae.png

New map. Taken from competitive play

mehwaffle10 committed:
Create SonantDread_Samsara.png

New map. Taken from competitive play

mehwaffle10 committed:
FG_Unearthed

Remove from dormant. Crush map horizontally. Raise and heavily adjust bedrock. Make it so defensive rotations are much harder by separating the flags with solid bedrock. Add a small structure and pre-dig tunnel to help offensive rotations. Remove void potential. Adjust resources and greenery. Lower map height. Pull red barrier closer to flags

mehwaffle10 committed:
Create mehwaffle10_Dustbowl.png

New map. Taken from competitive play

mehwaffle10 committed:
Update mapcycle.cfg

Add new maps

mehwaffle10 committed:
Update bunnie_courouz.png

grass

mehwaffle10 committed:
Grass

Fix missing grass

mehwaffle10 committed:
Update mehwaffle10_Dustbowl.png

Squish map horizontally. Adjust red barrier

mehwaffle10 committed:
Didgedy_TheHillsHaveEyes

Remove extra 1 from my mod

mehwaffle10 committed:
Update Hugobh_King_Gallery.png

grass

mehwaffle10 committed:
Update 4zK_Rorschach.png

grass

mehwaffle10 committed:
Renames

use usernames

mehwaffle10 committed:
Update mapcycle.cfg

Renames

mehwaffle10 committed:
Update TheCauldron.png

grass

mehwaffle10 committed:
Update slepovnickn_DrainedOcean.png

Grass

mehwaffle10 committed:
Update NoahTheLegend_MrSaucer.png

grass

mehwaffle10 committed:
Usernames

Enforce username_mapname naming convention. Use period for joint authorship, in alphabetical order

mehwaffle10 committed:
Joint Authorship

Have to use + instead of . since kag parses the extension off the first .

mehwaffle10 committed:
Create TerminalHash_Bridges_Above_Puddles.png

Initial commit. Accidentally already in mapcycle

mehwaffle10 committed:
TTH/WAR Folder Restructure

Restructure TTH folder to new format, add MM_ prefix to maps with his signature. Content unchanged, TTH needs maintenance first

mehwaffle10 committed:
Mapcycle Polishing

Mostly adjusting greenery. Fixed any weird tent or flag issues from symmetry

mehwaffle10 committed:
Greenery Tweaks

Handle all the weird cases that SonantDread's script flagged

Vam-Jam commented:
Merge conflicts will need to be sorted, I'd imagine you just want to override any changes with yours.

Vam-Jam commented on .gitignore:
```diff
@@ -0,0 +1,4 @@
+/Maps/CTF/test
+/Maps/TDM/New
+/Maps/SmallCTF/New
+/Maps/CTF/New
```
Is this .gitignore intentional?

mehwaffle10 replied:
No, it can be removed. Those were subfolders I dumped all the new maps into and pulled into the main area as they were completed. I also believe I changed Git to be case sensitive at one point

mehwaffle10 committed:
Delete .gitignore

mehwaffle10 committed:
Merge remote-tracking branch 'upstream/master' into mapcycle-update

mehwaffle10 committed:
Update HeraldsMarshes.png

fix shops

mehwaffle10 committed:
Update HeraldsMarshes.png

mehwaffle10 committed:
Update Skinney_RumbleGround.png

mehwaffle10 committed:
Update Skinney_RumbleGround.png

centered grain, clipped spawn

mehwaffle10 committed:
Update Ej_Chambers.png

Color mode, missing red barrier

mehwaffle10 committed:
Update Magmus_hangingmus.png

Fix bedrock holes due to slightly off color

mehwaffle10 committed:
Update 8x_BBiscuit.png

Raise map height

mehwaffle10 committed:
Update Monsteri_Sharcave.png

Push red barrier forward

mehwaffle10 committed:
Update MookMook_commerce.png

Fix asymmetrical flower
